core/domain/value_generators_domain.py

```diff
@@ -98,15 +104,15 @@ def _refresh_registry(cls):
                 cls.value_generators_dict[clazz.__name__] = clazz
 
     @classmethod
-    def get_all_generator_classes(cls):
+    def get_all_generator_classes(cls) -> Dict[str, BaseValueGenerator]:
+
         """Get the dict of all value generator classes."""
         cls._refresh_registry()
         return copy.deepcopy(cls.value_generators_dict)
 
     @classmethod
-    def get_generator_class_by_id(cls, generator_id):
+    def get_generator_class_by_id(cls, generator_id: str) -> BaseValueGenerator:
```

Comment:
This does not return a generator instance. This returns a generator class. You will need to use `Type[]` here, see https://mypy.readthedocs.io/en/stable/kinds_of_types.html#the-type-of-class-objects

Reply:
Done.